ts/deck-options/DeckOptionsPage.svelte

```diff
@@ -63,45 +64,64 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         --gutter-inline="0.25rem"
         --gutter-block="0.5rem"
         class="container-columns"
-        api={options}
     >
-        <Row class="row-columns">
-            <DailyLimits {state} api={dailyLimits} />
-        </Row>
-
-        <Row class="row-columns">
-            <NewOptions {state} api={newOptions} />
-        </Row>
-
-        <Row class="row-columns">
-            <LapseOptions {state} api={lapseOptions} />
-        </Row>
-
-        {#if state.v3Scheduler}
-            <Row class="row-columns">
-                <DisplayOrder {state} api={displayOrder} />
-            </Row>
-        {/if}
-
-        <Row class="row-columns">
-            <TimerOptions {state} api={timerOptions} />
-        </Row>
-
-        <Row class="row-columns">
-            <BuryOptions {state} api={buryOptions} />
-        </Row>
-
-        <Row class="row-columns">
-            <AudioOptions {state} api={audioOptions} />
-        </Row>
-
-        <Row class="row-columns">
-            <Addons {state} api={addonOptions} />
-        </Row>
-
-        <Row class="row-columns">
-            <AdvancedOptions {state} api={advancedOptions} />
-        </Row>
+        <DynamicallySlottable slotHost={Item} api={options}>
+            <Item>
+                <Row class="row-columns">
+                    <DailyLimits {state} api={dailyLimits} />
+                </Row>
+            </Item>
```

Comment:
I don't like the extra indentation at all, but it makes the approach considerably easier to understand imo.